mmdet/models/losses/gaussian_focal_loss.py

```diff
@@ -0,0 +1,59 @@
+import torch.nn as nn
+
+from ..builder import LOSSES
+
+
+def gaussian_focal_loss(pred,
+                        gaussian_target,
+                        weight=None,
+                        alpha=2.0,
+                        gamma=4.0):
+    eps = 1e-12
+    pos_weights = gaussian_target.eq(1)
+    neg_weights = (1 - gaussian_target).pow(gamma)
+    pos_loss = -(pred + eps).log() * (1 - pred).pow(alpha) * pos_weights
+    neg_loss = -(1 - pred + eps).log() * pred.pow(alpha) * neg_weights
+
+    if weight is not None:
```

Comment:
Use https://github.com/open-mmlab/mmdetection/blob/e37867530def6e935881a82f8d8861a1be3d09d9/mmdet/models/losses/utils.py#L26 for `weight`. 
Also, add `reduction`.